### DISCUSSION TITLE
有个关于udp的疑问

### DISCUSSION THREAD

**Discussion by LLIce** [kkkgo/PaoPaoGateWay#94] [问和答]
![QQ图片20240506122603](https://github.com/kkkgo/PaoPaoGateWay/assets/13741076/a19359f9-7b4f-439f-8a8f-4e57fd6d473a)
就用这么久了，好像就没看到过udp连接呢，不管是用自己节点还是机场节点，配置里udp也开了，不知道哪里不对劲。

**Comment by kkkgo** [comment]
udp本来就不常见啊，除非是quic或者DNS，游戏才会用到
